src/ctim/schemas/investigation.cljc

```diff
@@ -14,7 +14,12 @@
   c/sourced-object-entries
   c/describable-entity-entries
   (f/required-entries
-   (f/entry :type InvestigationIdentifier)))
+   (f/entry :type InvestigationIdentifier))
+  (f/optional-entries
+   (f/entry :object_ids (f/seq-of (f/str)))
+   ;; "type:value"
+   (f/entry :investigated_observables (f/seq-of (f/str)))
```

Comment:
Why do you prefer the string format for an observable? It will require additional parsing

Reply:
Originally, this is what I had in the initial commits. In a conversation with between @craigbro, @jbouwman, and I regarding these new fields and in the context of ES and CTIA, it was suggested the data for this field be stored as `"type:value"` rather than the full observable or a `{:type "value"}`. This is why the schema is this way. Personally, I have no preference; I'm just interested in doing the "right" thing.